src/encoded/static/components/browse/PublicationSearchView.js

```diff
@@ -5,12 +5,101 @@ import memoize from 'memoize-one';
-import { console } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
+import { console, analytics, object, navigate as spcNavigate } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
+import { TableRowToggleOpenButton } from '@hms-dbmi-bgm/shared-portal-components/es/components/browse/components/table-commons';
+
 import { columnExtensionMap } from './columnExtensionMap';
 import { Schemas } from './../util';
 
 import { transformedFacets } from './SearchView';
 
+
+const publicationColExtensionMap = _.extend({}, columnExtensionMap, {
+    // We override display_title for this view to be wider.
+    // And awesomer.
+    "display_title" : {
+        "widthMap" : { "sm" : 150, "md" : 300, "lg" : 520 },
+        "render" : function(result, columnDefinition, props, termTransformFxn, width){
+            return <PublicationSearchResultTitle {...props} {...{ result }} />;
+        }
+    }
+});
+
+
+class PublicationSearchResultTitle extends React.PureComponent {
+
+    /**
+     * Re-used for all titles - keep this in sync with CSS rules (line height, etc.)
+     * We could try to be smarter and like, roughly measure each character of alphabet or something, but... probably not worth it for now.
+     */
+    static maxCharsToShowPerLineBody = memoize(function(colWidth){
+        const charWidth = 6; // Rough upper estimate (excl "i", "l", ".", etc.)
+        const textAreaWidth = colWidth - 90; // Account for ToggleButton + cell padding, and line splitting on spaces (not in middle of words)
+        return Math.floor(textAreaWidth / charWidth);
+    });
+
+    static maxCharsToShowForTitle = memoize(function(colWidth){
+        const charWidth = 7; // Rough upper estimate (excl "i", "l", ".", etc.)
+        const textAreaWidth = colWidth - 90; // Account for ToggleButton + cell padding, and line splitting on spaces (not in middle of words)
+        return Math.floor(textAreaWidth / charWidth);
+    });
+
+    constructor(props){
+        super(props);
+        this.onClickTrack = this.onClickTrack.bind(this);
+    }
+
+    onClickTrack(evt){
+        const { result, href, navigate = spcNavigate } = this.props;
+        evt.preventDefault();
+        evt.stopPropagation();
+        analytics.productClick(result, {
+            'list'      : analytics.hrefToListName(href),
+            'position'  : rowNumber + 1
+        }, function(){
+            navigate(object.itemUtil.atId(result));
+        });
+        return false;
+    }
+
+    render(){
+        const { result, detailOpen, toggleDetailOpen, rowNumber, href, navigate = spcNavigate, width: colWidth } = this.props;
+        const { title: origTitle, "@id" : id, abstract, authors = null } = result; // We use "title" here, not "display_title" (which contains year+author, also)
+        const titleLen = abstract.length;
+        const titleMaxLen = PublicationSearchResultTitle.maxCharsToShowForTitle(colWidth) * 3;
+        let title = abstract;
```

Comment:
Just using abstract for testing. No long titles are present in test inserts and too lazy / not-enough-justifiable-time to create some.